### ISSUE TITLE
Initial validation do not store issues in ConstraintViolations

### ISSUE THREAD

**Issue by matek** [dooApp/FXForm2#96]
When FXForm is created, ConstraintViolations contains no issues even though validation errors are shown on form.
![fxform_issue](https://cloud.githubusercontent.com/assets/10861463/6056114/de4e26b2-ad0c-11e4-832a-4af769eaeeff.png)

Code:
```java

public class SaveMe extends Application {
	
	public class loginData {
		@NotNull @Length(min = 1, max = 8) 
	    private String name;

		@NotNull @Length(min = 1, max = 8) 
	    private String password;
		
		public String getName() {
			return name;
		}

		public void setName(String name) {
			this.name = name;
		}

		public String getPassword() {
			return password;
		}

		public void setPassword(String password) {
			this.password = password;
		}

	}
	
  @Override public void start(final Stage stage) throws Exception {
    final Label response = new Label();
    final Label desc = new Label("form.getConstraintViolations().size: ");
    final Button button = new Button("Save Me", imageView);
    loginData login = new loginData();

    FXForm<loginData> form = new FXForm<>(login);
    
    button.setContentDisplay(ContentDisplay.TOP);
    button.disableProperty().bind(Bindings.size(form.getConstraintViolations()).isNotEqualTo(0));
    response.textProperty().bind(Bindings.size(form.getConstraintViolations()).asString());
    layoutScene(form, button, desc, response, stage);
    stage.show();
  }
 
 private void layoutScene(final Node content, final Button button, final Label desc, final Label response, final Stage stage) {
    final VBox layout = new VBox(10);
    layout.setPrefWidth(500);
    layout.setAlignment(Pos.CENTER);
    layout.getChildren().addAll(content, button, desc, response);
    layout.setStyle("-fx-background-color: cornsilk; -fx-padding: 20; -fx-font-size: 20;");
    stage.setScene(new Scene(layout));
  }

  public static void main(String[] args) { launch(args); }
}
```

**Comment by amischler** [comment]
Fixed in 8.0.8-SNAPSHOT

**Comment by amischler** [comment]
Fixed in 8.0.8
